### DISCUSSION TITLE
Velocity Curve Activity Question

### DISCUSSION THREAD

**Discussion by hiltonml** [WVURAIL/dspira-lessons#10] [Astronomy]
In the teacher notes for the velocity curve activity, part 1 it is mentioned that depending on the time of the year, the quadrant II peaks may not all be blue-shifted and the quadrant III peaks may not all be red-shifted.  What time of year is this?  Or does it vary from year to year?

Thank you,
Mike Hilton

**Comment by jmakous** [comment]
The orbital motion of the earth around the sun causes a change in the relative motion of different parts of the galaxy relative to earth. So a part of the galaxy for which the shift is only slightly red-shifted or slightly blue-shifted, relative to the sun, can become blue- or red-shifted, respectively, at different times of the year. The earth’s speed around the sun is approximately 29 km/s, and the earth’s orbital motion results in a sinusoidal velocity shift relative to the sun. So, for instance, a part of the MWG that is always slightly red-shifted relative to the sun will be more red-shifted at certain times of the year, and then become blue-shifted at other times of the year.

The lesson titled [”Measuring the Earth's Speed around the Sun”](https://wvurail.org/dspira-lessons/EarthAroundSun) under the DSPIRA-Lessons Portal has a good activity to do with the students. There is a nice sinusoidal graph under the teacher notes showing the relative motion of the earth.

**Comment by hiltonml** [comment]
Thank you very much for your reply.  This data explains the discrepancy
between my class' doppler shift measurements and the "theoretical" results
we expected.

Mike Hilton, Ph.D. <https://www.eckerd.edu/computer-science/faculty/hilton/>
Assistant Professor of Computer Science
727.864.8272
Pronouns: he/him/his

Eckerd College <http://www.eckerd.edu>
4200 54th Avenue South
St. Petersburg, Florida 33711
[image: Eckerd College logo]




*Instructions for living a life:Pay attention.Be astonished.Tell about it.*
*-- *Mary Oliver, *Sometimes*
